web

```diff
@@ -1 +0,0 @@
-Subproject commit 83463c2bcd5c4360adbc41c61e6572406fec54ce
```

Comment:
@solderzzc to always get master version of the web submodule I think. But I'm not sure.